src/main/java/com/datapipe/jenkins/vault/credentials/VaultKubernetesCredential.java

```diff
@@ -21,11 +21,14 @@
     @NonNull
     private final String role;
 
+    private final String mountPath;
+
     @DataBoundConstructor
     public VaultKubernetesCredential(@CheckForNull CredentialsScope scope, @CheckForNull String id,
-        @CheckForNull String description, @NonNull String role) {
+        @CheckForNull String description, @NonNull String role, String mountPath) {
```

Comment:
add it as a databound setter instead

Reply:
I've never written a jenkins plugin before but I followed https://jenkins.io/doc/developer/plugin-development/pipeline-integration/#constructor-vs-setters

Is what i've just done correct?

Reply:
yup 👍